pinot-connectors/pinot-flink-connector/pom.xml

```diff
@@ -51,7 +51,18 @@
     </dependency>
     <dependency>
       <groupId>org.apache.flink</groupId>
-      <artifactId>flink-streaming-java_${scala.compat.version}</artifactId>
```

Comment:
whats the scala version required for flink 1.19?

Reply:
As far as I can tell, Flink 1.14.x used a Scala version suffix on the artifact name:

https://mvnrepository.com/artifact/org.apache.flink/flink-streaming-java_2.12/1.14.3

Flink 1.19.0 does not have a Scala suffix on the artifact name:

https://mvnrepository.com/artifact/org.apache.flink/flink-streaming-java/1.19.0

Reply:
right, but does it drop any scala version support? if so, we need to mark in the release notes about the back-compat break

Reply:
@yupeng9 it looks that Flink 1.15 dropped the support for `scala 2.11`. We have been using scala 2.12 from Pinot code base. 

I think that this change looks fine to me. How do you think?

https://stackoverflow.com/questions/70792112/what-version-of-scala-does-apache-flink-use

Reply:
At uber we use 2.12 too. but it'll be good to call out any breaking changes in the release